llvm/test/tools/llvm-objdump/XCOFF/disassemble-symbol-priority.ll

```diff
@@ -4,7 +4,10 @@
 ; CHECK: Disassembly of section .text:
 ; CHECK: 00000000 <.foo3>:
 ; CHECK: 00000020 <.foo4>:
-; CHECK: 00000040 <.foo>:
+
+;; FIXME: should show the function name instead of the section name.
+; CHECK: 00000040 <explicit_sec>:
```

Comment:
Since @diggerlin you pointed out the prioritizing policy for symbols in llvm-objdump, could you please help to have a check for this case? Seems when SD symbol functions are added in, llvm-objdump no longer gets a right priority for this case. If you won't to do so or no time at this moment, I'll handle them later.

```
[7]     m   0x00000040     .text     1  unamex                    explicit_sec
[8]     a4  0x0000003e       0    0     SD       PR    0    0
[9]     m   0x00000040     .text     1  extern                    .foo
[10]    a4  0x00000007       0    0     LD       PR    0    0
```

Seems if `LD` symbols have higher priorities than `SD` symbols, llvm-objdump should print `.foo` instead. No sure why it generates wrong result.

Reply:
your need to add option for `--symbol-description` for llvm-objdump , otherwise changing to CHECK: 00000040 <explicit_sec> is correct.

suggest add `--symbol-description` into the test case.